kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/orders/ordering/OrderProcessResultDto.java

```diff
@@ -0,0 +1,49 @@
+package com.kodilla.good.patterns.orders.ordering;
+
+public final class OrderProcessResultDto {
```

Comment:
In my opinion in module there is a huge error about DTO: see definition on wikipedia: https://en.wikipedia.org/wiki/Data_transfer_object
DTO is an object that carries data between processes (for example DTO can be object send via network)
So it is rather business object and name should be OrderProcessResult